src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/NetworkListener.java

```diff
@@ -39,4 +45,12 @@ public void onBlockBreak(BlockBreakEvent e) {
     public void onBlockPlace(BlockPlaceEvent e) {
         manager.updateAllNetworks(e.getBlock().getLocation());
```

Comment:
unrelated to this PR but surely this can be optimised. Placing will only ever affect the network if it's a SF block. Looking at the method it looks pretty intense.

Why does the manager even have a List rather than a Set ughh, I really hate the way this is done currently.